### ISSUE TITLE
How to build 32x windows dll with mingw-w64

### ISSUE THREAD

**Issue by MCRusher** [wmcbrine/PDCurses#83]
Tried calling `make CFLAGS="-m32" DLL=Y`, linking fails with a lot of undefined symbols.

tried editing CC and LINK in the makefile to use `-m32`, same result.

Need to use it with a 32 bit compiler, but it refuses to build in anything but 64 bits.

Thanks for any help.

**Comment by wmcbrine** [comment]
When I installed mingw-w64, I installed a 32-bit version (i686-8.1.0-posix-dwarf-rt_v6-rev0), alongside a 64-bit version (x86_64-8.1.0-posix-seh-rt_v6-rev0). Each of these builds it natively, with no extra flags needed. I haven't really explored cross-compiling it.

**Comment by Bill-Gray** [comment]
I _have_ explored cross-compiling.  In fact,  I rarely do Windows builds any other way.  [The Makefile from my fork](https://github.com/Bill-Gray/PDCurses/tree/master/wincon) will work with William's branch.  For cross-compiling,  you need to add either `_w32=1` or `_w64=1` to get the suitable prefix.
It essentially does what you describe (adding prefixes to `CC`,  `LINK`,  `LIBEXE`,  and `WINDRES`).  Maybe you missed one?

**Comment by MCRusher** [comment]
Ok so I took forever to fix it, I felt bad about leaving it open but I didn't want to close it without resolving the issue.

I made manual changes to the values to now be

`CC      = i686-w64-mingw32-gcc
AR		= i686-w64-mingw32-gcc-ar
STRIP		= strip
LINK		= i686-w64-mingw32-gcc
WINDRES		= C:\mingw-w64\i686-8.1.0-win32-sjlj-rt_v6-rev0\mingw32\bin\windres`

and it seems to work.

I reinstalled mingw both 64 and i686 and also realized I had been using the wrong make, instead of mingw32=make.
I fixed those but mingw32-make was still calling the wrong gcc so I added both to path and referenced the i686 specific one.
Now it seems to build fine.

Thanks for listening and trying to help, and sorry I took so long to finish.
